JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how the **Chart** widget aggregates series points in&nbsp;a&nbsp;financial chart. The **useAggregation** option activates this feature. The **RangeSelector** widget zooms the contents of&nbsp;the **Chart**. Note that whenever the selected range is&nbsp;reduced, the series becomes less aggregated.
\ No newline at end of file
+The **Chart** widget can aggregate data represented by series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
+
+In this demo, the chart's [visual range](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/visualRange/) depends on the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/)'s settings (see the [onValueChanged](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/Configuration/#onValueChanged) handler). When the selected range is reduced, the series automatically becomes less aggregated. To change this behavior and set the fixed aggregation degree, specify the [aggregationInterval](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/aggregationInterval/) option of an axis.
```

Comment:
It is better to being from the beginning:

```suggestion
In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. When you narrow down the selection range, the chart zooms in, and the series points become less aggregated. The converse is also true: when the selected range is wider, the degree of aggregation is higher.
```

I also removed the last sentence for two reasons: 1) `aggregationInterval` is not the only means of controlling the degree of aggregation - there's also `aggregationGroupWidth`; 2) this demo has an option called `aggregationInterval` set for the RangeSelector, and it is not the same option. I think it is better to send the user to the `aggregation` description and let him find out the details from there. Hence, the suggestion for the paragraph below.